tests/acceptance/Makefile.am

```diff
@@ -87,6 +87,7 @@ endif
 # to run in parallel; TODO fix "make -n" not working:
 check-local:
 	+ $(TESTS_ENVIRONMENT) MAKE=$(MAKE) $(srcdir)/testall
+	+ $(srcdir)/selftest.sh
```

Comment:
I put this second so that the other tests would always run.

Sadly I didn't think it was worth the effort to generate an XML JUnit report for selftest.sh. Maybe another ticket for clarity in CI. :) As-is a non-zero exit code will occur if there is a problem and the log should be fairly easy to read with lots of `error:` prefixed lines and complete log output included. (so 1/2 batteries are included with this PR)

Reply:
Sounds good enough to me.